client/public/index.html

```diff
@@ -1 +1,10 @@
-<div id="root"></div>
\ No newline at end of file
+<!DOCTYPE html>
```

Comment:
idk how firebase hosting munges their html, but I put this here for now @swalschuler